handlers/admin_handler.py

```diff
@@ -113,25 +175,18 @@ def execute(cls, slack_wrapper, args, timestamp, channel_id, user_id, user_is_ad
 class AdminHandler(BaseHandler):
     """
     Handles configuration options for administrators.
-
-    Commands :
-    # Show administrator users
-    !admin show_admins
-
-    # Add a user to the administrator group
-    !admin add_admin user_id
-
-    # Remove a user from the administrator group
-    !admin remove_admin user_id
     """
 
     def __init__(self):
         self.commands = {
             "show_admins": CommandDesc(ShowAdminsCommand, "Show a list of current admin users", None, None, True),
             "add_admin": CommandDesc(AddAdminCommand, "Add a user to the admin user group", ["user_id"], None, True),
             "remove_admin": CommandDesc(RemoveAdminCommand, "Remove a user from the admin user group", ["user_id"], None, True),
-            "as": CommandDesc(AsCommand, "Execute a command as another user", ["@user", "command"], None, True)
+            "as": CommandDesc(AsCommand, "Execute a command as another user", ["@user", "command"], None, True),
+            "maintenance": CommandDesc(ToggleMaintenanceModeCommand, "Toggle maintenance mode", None, None, True),
+            "debug": CommandDesc(StartDebuggerCommand, "Break into a debugger shell", None, None, True),
+            "join": CommandDesc(JoinChannelCommand, "Join a channel", ["channel_name"], None, True),
+            "makectf": CommandDesc(MakeCTFCommand, "Turn the current channel into a CTF channel by setting the purpose. Requires reload to take effect", ["ctf_name"], None, True)
```

Comment:
Why not move the makectf command to the challenge handler and directly create a ctf object and add it to the challenge db as the addctf command would do, so you don't have to reload? Also, since the command is ctf/challenge specific it would belong to the challenge-handler anyways.

Reply:
Let's file a ticket for this. We want to merge 'as in' to sync up with their repo.